search/views.py

```diff
@@ -186,7 +188,9 @@ def course_discovery(request):
                 "search_term": search_term,
                 "page_size": size,
                 "page_number": page,
-                "catalog_visibility": catalog_visibility
+                "catalog_visibility": catalog_visibility,
+                "sort": sort,
```

Comment:
Please clarify why do we need to update track.emit dict

Reply:
This dictionary is used for logging. These logs are used by analytics. I think we need to add these params to log file.
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/analytics.html